pkg/schemas/model.go

```diff
@@ -73,9 +73,9 @@ type Type struct {
 	Ref     string `json:"$ref,omitempty"`    // section 7
 	// RFC draft-wright-json-schema-validation-00, section 5
 	MultipleOf           int              `json:"multipleOf,omitempty"`           // section 5.1
-	Maximum              int              `json:"maximum,omitempty"`              // section 5.2
+	Maximum              float64          `json:"maximum,omitempty"`              // section 5.2
 	ExclusiveMaximum     bool             `json:"exclusiveMaximum,omitempty"`     // section 5.3
```

Comment:
Exclusives are not following the spec but I just ignored that issue, could be fixed though :D

Reply:
solved now